test/index.js

```diff
@@ -460,4 +470,81 @@ describe('KeyringController', function () {
       );
     });
   });
+
+  describe('submitEncryptionKey', function () {
```

Comment:
Maybe `cacheEncryptionKey` is a better description of this section? Only a few tests are about the `submitEncryptionKey` function, but they all seem to relate to this option.